packages/@aws-cdk/cfnspec/build-tools/update-cfnlint.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -euo pipefail
+scriptdir=$(cd $(dirname $0) && pwd)
+
+# Download (parts of) the cfn-lint repo that we use to enhance our model
+intermediate="$(mktemp -d)/tmp.zip"
+
+url="https://github.com/aws-cloudformation/cfn-python-lint/archive/master.zip"
+echo >&2 "Downloading from ${url}..."
+curl -sSfL "${url}" -o ${intermediate}
+
+for file in StatefulResources; do
```

Comment:
minor: not familiar with this bash-ism: what is `StatefulResources`?

Reply:
A literal string :D